packages/react-badge/Spec.md

```diff
@@ -0,0 +1,118 @@
+# [Badge](https://github.com/microsoft/fluentui/issues/16925)
```

Comment:
I agree with @behowell there:
https://github.com/microsoft/fluentui/pull/16644#issuecomment-776888522

Let's keep spec separate until it will be merged to master